components/ws-daemon/cmd/run.go

```diff
@@ -130,32 +74,25 @@ var runCmd = &cobra.Command{
 
 		go config.Watch(configFile, dmn.ReloadConfig)
 
-		// run until we're told to stop
-		sigChan := make(chan os.Signal, 1)
-		signal.Notify(sigChan, os.Interrupt, syscall.SIGTERM)
-		log.Info("🧫 ws-daemon is up and running. Stop with SIGINT or CTRL+C")
-		<-sigChan
-		server.Stop()
-		err = dmn.Stop()
+		err = srv.ListenAndServe()
 		if err != nil {
-			log.WithError(err).Error("cannot shut down gracefully")
+			log.Fatal(err)
```

Comment:
```suggestion
			log.WithError(err).Fatal("Failed to listen")
```

This prints the error structured the same as other errors, easier for log aggregators